parm/config/config.resources

```diff
@@ -191,6 +192,80 @@ elif [[ "${step}" = "atmanalpost" ]]; then
     export npe_node_atmanalpost
     export is_exclusive=True
 
+elif [[ "${step}" = "landanlinit" ]]; then
+
+   # below lines are for creating JEDI YAML
+   case ${CASE} in
+     C768)
+        layout_x=6
+        layout_y=6
+        ;;
+      C384)
+        layout_x=5
+        layout_y=5
+        ;;
+     C192 | C96 | C48)
+        layout_x=8
+        layout_y=8
+        ;;
+      *)
+          echo "FATAL ERROR: Resolution not supported for land analysis'"
+          exit 1
+    esac
+
+    export layout_x
+    export layout_y
+
+    export wtime_landanlinit="00:10:00"
+    export npe_landanlinit=1
+    export nth_landanlinit=1
+    npe_node_landanlinit=$(echo "${npe_node_max} / ${nth_landanlinit}" | bc)
+    export npe_node_landanlinit
+    export memory_landanlinit="3072M"
+
+elif [[ "${step}" = "landanlrun" ]]; then
+
+   case ${CASE} in
+     C768)
+        layout_x=6
+        layout_y=6
+        ;;
+      C384)
+        layout_x=5
+        layout_y=5
+        ;;
+     C192 | C96 | C48)
+        layout_x=8
+        layout_y=8
+        ;;
```

Comment:
I know this is just an initial PR, but seems weird for the grid decomposition to increase with coarser resolution. These coarse resolution are key for workflow development to keep the resources down to allow rapid testing.

Reply:
@WalterKolczynski-NOAA Thanks for findings, and I made changes from `8` to `1` for the coarser resolution.